source/frontend/src/features/documents/ComposedDocument.ts

```diff
@@ -123,7 +123,7 @@ export class BatchUploadResponseModel {
         this.errorMessage = (apiResponse as IApiError).details;
       } else {
         this.isSuccess =
-          apiResponse.uploadResponse?.documentExternalResponse?.status === 'Success' ?? false;
+          apiResponse.uploadResponse?.documentExternalResponse?.status === 'Success' ? true : false;
```

Comment:
this change doesn't seem related to the PR - could you please take it out?

Reply:
it's a warning from Vite. since left had operator always returns true.